recipes/kivy/build.sh

```diff
@@ -0,0 +1,8 @@
+if [ "${TRAVIS_OS_NAME}" == "osx" ]; then
+  ln -s /usr/lib/x86_64-linux-gnu/libGL.so $PREFIX/lib/libGL.so;
```

Comment:
Pretty sure we don't want this on macOS. Maybe it is useful on Linux, but I don't think it should be needed there either.

Reply:
That was a mistake and should have read linux, not osx. But since it didn't error out on linux, I guess it's not needed. I added it because when I built it on ubuntu on my machine, it errored out saying that it couldn't link with lGL. Adding that link fixed it for me.

I'm trying a new build without it to see if it fails. I also temporarily disabled trying to find gstreamer on osx until the pinning pr is merged, so I can see if it would fail to the gl link not being there.